cc33calibrator/suite/trixie/debian/rules

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --sourcedir=cc33calibrator
+
+override_dh_auto_build:
+	dh_auto_build -- CFLAGS="$(CFLAGS) -I/usr/include/libnl3/ -DCONFIG_LIBNL32" LDFLAGS="$(LDFLAGS) -L/usr/lib/" "NLVER=3"
```

Comment:
None of this should be needed now that `cc33calibrator` is also a CMake project.